.github/workflows/release_components.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           registry-url: 'https://registry.npmjs.org'
 
       - name: Version ui-components
-        run: yarn version:prerelease:ui-components
```

Comment:
curious what specific things this will change re: uptakes? e.g. no more alpha versions?

Reply:
Correct. There will be no more alpha version. It should (if this works correct) just always do major/minor/patch versions